pkg/alert/ctl_alert.go

```diff
@@ -131,6 +134,9 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.Registry, "registry", ctl.Registry, "Name of the registry to use for images e.g. docker.io/blackducksoftware")
 	cmd.Flags().StringSliceVar(&ctl.PullSecrets, "pull-secret-name", ctl.PullSecrets, "Only if the registry requires authentication")
 	cmd.Flags().StringSliceVar(&ctl.ImageRegistries, "image-registries", ctl.ImageRegistries, "List of image registries")
+	cmd.Flags().StringVar(&ctl.CertificateFilePath, "certificate-file-path", ctl.CertificateFilePath, "Absolute path to a file for an Alert nginx certificate")
+	cmd.Flags().StringVar(&ctl.CertificateKeyFilePath, "certificate-key-file-path", ctl.CertificateKeyFilePath, "Absolute path to a file for an Alert nginx certificate key")
+	cmd.Flags().StringVar(&ctl.CustomCAFilePath, "ca-certificate-file-path", ctl.CustomCAFilePath, "Absolute path to a file for the CA certificate for Alert")
```

Comment:
The description should be updated. It is not expecting a CA certificate but a java keystore.

Reply:
fixed